sync/src/com/kirinpatel/gui/ControlPanel.java

```diff
@@ -54,16 +54,16 @@ private ControlPanel(GUI gui) {
             public Component getListCellRendererComponent(JList list, Object value, int index, boolean isSelected, boolean cellHasFocus) {
                 Component c = super.getListCellRendererComponent(list, value, index, isSelected, cellHasFocus);
 
-                if (index < Main.connectedUsers.size()) {
-                    final User host = Main.connectedUsers.get(0);
-                    final User user = Main.connectedUsers.get(index);
+                if (index < sync.connectedUsers.size()) {
+                    final User host = sync.connectedUsers.get(0);
```

Comment:
can we make an optional<user> field in sync for host so this can read sync.host.get()